platform/mwm_traits.cpp

```diff
@@ -4,23 +4,29 @@
 
 namespace version
 {
-MwmTraits::MwmTraits(version::Format versionFormat) : m_versionFormat(versionFormat) {}
+MwmTraits::MwmTraits(MwmVersion const & version) : m_version(version) {}
 
 MwmTraits::SearchIndexFormat MwmTraits::GetSearchIndexFormat() const
 {
-  if (m_versionFormat < version::Format::v7)
+  if (m_version.GetFormat() < version::Format::v7)
```

Comment:
Up to you, but you can implement a couple of helper getters in the private section, say, GetFormat() and GetVersion(), and call them here instead of this usage of m_version.